service/models.py

```diff
@@ -113,4 +113,31 @@ def init_db(cls, app):
     def all(cls):
         """ Returns all of the Recommendations in the database """
         cls.logger.info('Processing all Recommendations')
-        return cls.query.all()
\ No newline at end of file
+        return cls.query.all()
+
+    #@classmethod
```

Comment:
you should remove commented codes from PR to keep the code base clean.

Reply:
That's my fault, I have deleted it.